o4/o4.py

```diff
@@ -1570,7 +1575,7 @@ def main():
             if opts['--report']:
                 print(opts['--report'].format(**locals()))
         if opts['status']:
-            o4_status(opts['@'], opts['<path>'], opts['-f'], opts['-q'])
+            ec = o4_status(opts['@'], opts['<path>'], opts['-f'], opts['-q'])
```

Comment:
That becomes weird if you have exit code 672890.

Reply:
good point